libs/barista-components/secondary-nav/src/section/secondary-nav-section.ts

```diff
@@ -78,40 +78,21 @@ export class DtSecondaryNavSection {
       this._changeDetectorRef.markForCheck();
     }
   }
+  /** Whether the section is expanded. */
   private _expanded = false;
 
-  /** Whether the section is expandable. */
-  @Input()
-  get expandable(): boolean {
-    return this._expandable;
-  }
-  set expandable(value: boolean) {
-    this._expandable = coerceBooleanProperty(value);
-  }
-  private _expandable = false;
+  /** Destroy subject that fires when the component is destroyed. */
+  private _destroy$ = new Subject<void>();
 
-  /** Whether the section is external. */
-  @Input()
-  get external(): boolean {
-    return this._external;
-  }
-  set external(value: boolean) {
-    this._external = coerceBooleanProperty(value);
-  }
-  private _external = false;
+  /** @internal Whether the section is active. */
+  _active = false;
 
-  /** Whether the section is active. */
-  @Input()
-  get active(): boolean {
-    return this._active;
-  }
-  set active(value: boolean) {
-    this._active = coerceBooleanProperty(value);
-    if (this._active && this._expandable) {
-      this.expanded = true;
-    }
-  }
-  private _active = false;
+  /** @internal List of all groups within the section. */
```

Comment:
If the property is private there is no need to annotate it as internal.
```suggestion
  /** List of all groups within the section. */
```